src/CMakeLists.txt

```diff
@@ -22,7 +22,9 @@ add_executable(target-isns
   log.c
   util.c
   target-isns.c)
-set_property(TARGET target-isns PROPERTY C_STANDARD 99)
+if(IS_CMAKE_31)
```

Comment:
You can drop this because CMake versions older than 3.1 do not complain (at least CMake 2.8 does not complain).